Verification/FLiBe_Prop_Verification.py

```diff
@@ -0,0 +1,180 @@
+# Sierra Tutwiler
+# Last Modified: 5/22/2020
+# Verify fluid properties for Flibe
+# Reference: "Annals of Nuclear Energy", Romatoski and Hu
+# Note: 
+	#Temperature is in Kelvin
+
+import numpy as np 
+import matplotlib.pyplot as plt  
+import FLiBe_Prop
+import Flibe_Cp_Verification_Data
+import pandas as pd
+
+steps = 700
+
+
+T = np.linspace(550,1250,steps)
+T2 = np.linspace(700,1200,steps)
+
+Rho = np.zeros(steps)
+
+Nu = np.zeros(steps)
+
+Cp = np.zeros(steps)
+
+K = np.zeros(steps)
+
+
+for i in range(0,steps):
+	Rho[i] = FLiBe_Prop.rho(T[i])
+
+for i in range(0,steps):
+	Nu[i] = FLiBe_Prop.nu(T2[i])*1000
+
+for i in range(0,steps):
+	Cp[i] = FLiBe_Prop.Cp(T2[i])
+
+for i in range(0,steps):
+	K[i] = FLiBe_Prop.k(T2[i])
+
+data1 = pd.read_csv(r'C:\Users\Sierra\PycharmProjects\Giraffe\TACOCAT\Verification\Flibe\Density\Romatoski_Flibe_Density_Blanke_1956.csv')
```

Comment:
Let's talk about how to make this generic. There is a way to import sys so it adds this without hardcoding it.